test/kubectl-layer.integ.ts

```diff
@@ -2,20 +2,37 @@ import * as path from 'path';
 import * as cdk from 'aws-cdk-lib';
 import * as lambda from 'aws-cdk-lib/aws-lambda';
 import * as cr from 'aws-cdk-lib/custom-resources';
+import { IConstruct } from 'constructs';
 
-import { KubectlV29Layer } from '../lib';
+import { KubectlV30Layer } from '../lib';
 
 /**
- * Test verifies that kubectl and helm are invoked successfully inside Lambda runtime.
+ * override the framework.onEvent lambda function runtime from nodejs14.x to nodejs16.x
+ * as nodejs14.x is deprecated.
  */
+function overrideProviderRuntime(construct?: IConstruct[]) {
```

Comment:
instead of this shouldn't we update min cdk version to a version that doesnt have node14 as the framework runtime?

Reply:
@awsdataarchitect yes your PR is for 1.29.0 which is still relevant. My PR is for 1.30 branch though.